draft-ietf-taps-impl.md

```diff
@@ -1011,7 +1011,7 @@ Close:
 : Calling `Close` on a UDP Connection (ABORT.UDP(-Lite)) releases the local port reservation.
 
 Abort:
-: Calling `Abort` on a UDP Connection (ABORT.UDP(-Lite)) is identical to calling `Close`.
+: Calling `Abort` on a UDP Connection (ABORT.UDP(-Lite)) is identical to calling `Close`except that a ConnectionErrror Event rather than a Closed Event will be sent by the Connection.
```

Comment:
```suggestion
: Calling `Abort` on a UDP Connection (ABORT.UDP(-Lite)) is identical to calling `Close`, except that the Connection will send a ConnectionError Event rather than a Closed Event.
```